lib/constants.js

```diff
@@ -13,77 +13,95 @@ export const FONTS = [
 export const THEMES = [
   {
     id: '3024-night',
-    name: '3024 Night'
+    name: '3024 Night',
+    background: 'rgba(9, 3, 0, alpha) !important'
```

Comment:
I am thinking for this you can just look at the dom-node's background color dynamically and adjust the opacity as you have done here. That way if people want to add a new theme, they can do so a lot easier than manually setting an `rgba`.

Reply:
Makes sense. Will make the changes (y)